src/components/App.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import MenuBar from './MenuBar.jsx' ;
+import PlayGrid from './PlayGrid.jsx' ;
+
+const ROWS = 10;
+const COLS = 10;
+
+
+function create2DArray(ROWS, COLS){
```

Comment:
I would prefer `rowCount` and `colCount` instead of `UPPER_CASE` argument names. These arguments can vary between function calls. They're not exactly constants.